Resources/views/block_item/product.html.twig

```diff
@@ -0,0 +1,21 @@
+<article class="selected-products">
+    <div class="attribute-image">
+        <a href="{{ path( location ) }}">
+            {{ ez_render_field(
+                content, "image",
+                {
+                    "template": "eZDemoBundle:fields:ezimage_simple.html.twig",
+                    "parameters": {"alias": "productthumbnail"}
+                }
+            ) }}
+        </a>
+    </div>
+    <div class="product-info">
+        <div class="attribute-header">
+            <a href="{{ path( location ) }}" class="teaser-link">
+                <h3>{{ ez_content_name( content ) }}</h3>
+            </a>
+        </div>
+    </div>
+</article>
+
```

Comment:
so, we aren't showing price nor submit button here? i guess this is because shop still relies on legacy, right?